content/posts/euc/hub-templates.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "Hub Service Template"
+date: 2020-10-24T00:00:00-00:00
+draft: false
+categories: [HUBSERVICES, EUC]
+author: Nagul Subramanian
+authorlink: /https://www.linkedin.com/in/nagulsubramanian 
+tags:
+- Access
+- Hub Services
+- Templates 
+- Workspace ONE
+lightgallery: true
+---
+
+## What is Hub Services? 
+To simply state it helps enterprise introduce a true digital experience to their employees on any device or web. This is just getting started with lot of potential to unlock features to make their employee experience a world class. 
+
```

Comment:
I think it'll be good to provide more context around what Hub Services is and what capabilities it enabled for those that are not aware.